data/Wooller_2002/data.csv

```diff
@@ -0,0 +1,4 @@
+﻿Species,resprouting_capacity,reproductive_maturity,fire_time_from_fire_to_flowering,fire_time_from_fire_to_50%_flowering,fire_time_from_fire_to_fruit,fire_time_from_fire_to_50%_fruit,seedbank_location,recruitment_timing,post_fire_recruitment,serotiny,plant_height_max
+Banksia nutans,fire_killed,6,6,20,6,15,canopy_seedbank,YYNNNNNNNNNY,post_fire_recruitment,serotiny_moderate,4
```

Comment:
Where did the `serotiny_moderate` values come from?

Reply:
Do you think it would be better to simply use "serotinous" - that is what I'm now leaning towards

Reply:
I think so, unless you found somewhere in the manuscript where they describe the level of serotiny